src/models/tiles/table/table-content.ts

```diff
@@ -299,6 +299,45 @@ export const TableContentModel = TileContentModel
             sharedDataSet.dataSet = DataSet.create(getSnapshot(self.importedDataSet));
             self.clearImportedDataSet();
           }
+
+          // Originally this code was in doPostCreate. When shared datasets were implemented in
+          // CLUE 3.0.0 this code was disabled.
+          // - The `attr.formula.synchronize` is skipped, because it would add a new undo/redo entry.
+          //   I think this synchronizing was here to handle backwards compatibility. Since it hadn't
+          //   been running for many months it seems safe to skip it. There is a chance that some old
+          //   document with an out of sync formula might be loaded. I'm not sure what will happen in
+          //   that case.
+          // - the updateDatasetByExpressions is skipped for the same reason. It would modify the dataset
+          //   which would generate multiple undo/redo entries. Currently it seems the table is responsible
+          //   for computing the formula and updating the dataset values. Because the table formula code
+          //   was broken for a while, there will be documents where the values are out of sync with the
+          //   formula. Without the updateDatasetByExpressions these out of sync values will not be
+          //   updated.
+          // A change to make here is to move this out of metadata just use volatile on the table
+          // content model. And possibly even better would be to move the formula calculation out of the
+          // table.
```

Comment:
@pjanik's current work on CODAP v3 adds an independent formula-evaluation system that's not tied to any one component.